packages/expo-modules-core/ios/Swift/Modules/Uuid/Uuidv5.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import CommonCrypto
+import CryptoKit
+func uuidv5(name: String, namespace: UUID) -> UUID {
```

Comment:
```suggestion

func uuidv5(name: String, namespace: UUID) -> UUID {
```

Reply:
@lukmccall Hmm this is weird, I commited code that contains this line (the blank space). It shows up in my local code and in the github web editor (press dot on the keyboard to open). I'm not sure why it's not visible in here 🧐🤔